libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -306,6 +306,10 @@
 #define HAL_CANFD_SUPPORTED 0
 #endif
 
+#ifndef DISABLE_DSHOT
```

Comment:
Like @peterbarker I really, really wish we could change this very counter-intuitive name